newIDE/app/src/ObjectsRendering/PixiResourcesLoader.js

```diff
@@ -209,4 +210,17 @@ export default class PixiResourcesLoader {
   static getInvalidPIXITexture() {
     return invalidTexture;
   }
+
+    /**
```

Comment:
Align this comment/run autoformatting everywhere please. For every commit, remember to run it.